static/openapi/vwrs-spec.json

```diff
@@ -1164,6 +1228,194 @@
         }
       }
     },
+    "/api/vwr/v1/requests/status/{request_id}": {
+      "get": {
+        "summary": "Position in waiting room",
+        "description": "Get the user's current position in waiting room.",
+        "tags": [
+          "Waiting room"
+        ],
+        "security": [
+          {
+            "APIKeyAuth": []
+          }
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "request_id",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The unique request ID of a user."
+          },
+          {
+            "in": "query",
+            "name": "domain",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The domain_key of the domain."
+          },
+          {
+            "in": "query",
+            "name": "sid",
+            "required": true,
+            "schema": {
+              "type": "integer"
+            },
+            "description": "The position ID in the waiting room."
+          },
+          {
+            "in": "query",
+            "name": "created_at",
+            "required": true,
+            "schema": {
+              "type": "integer"
+            },
+            "description": "The creation timestamp when user first time entered the waiting room."
+          },
+          {
+            "in": "query",
+            "name": "client_version",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The version of the EdgeWorker."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully fetched the user's position status.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/GetStatusResponse"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad request."
+          },
+          "401": {
+            "description": "Authorization failure due to invalid authentication credentials."
+          },
+          "500": {
+            "description": "Internal server error."
+          }
+        }
+      }
+    },
+    "/api/vwr/v1/position/{domain_key}": {
+      "get": {
+        "summary": "Provide initial position in the waiting room",
+        "description": "Provide initial position in the waiting room.",
```

Comment:
Why "provide" rather than "get," which is used earlier?

Reply:
Yes.  Get.